### DISCUSSION TITLE
Home page: left sidebar versus main content

### DISCUSSION THREAD

**Discussion by jmfayard** [community/community#34492] [Other Feature Feedback, Questions, & Ideas]
Bonjour everyone,
I'm not super convinced by the homepage if the goal is to actually focus on your work

The left sidebar is super relevant to my work and I would like to focus on that
The rest is mostly noise

![image](https://user-images.githubusercontent.com/459464/192880403-4cf8a848-1b8c-4320-8131-531a6eab6d6c.png)

**Comment by jeaolu** [comment]
Well you got a nice profile

**Comment by kaushikravuri** [comment]
That's cool and good to you too

**Reply by jeaolu** [reply]
Thank you 😊
